### PR TITLE
Documentation: Add documentation covering how to install using the Openshift OperatorHub

### PR DESCRIPTION
**Pull request by chancez**
Rendered version: https://github.com/chancez/operator-metering/blob/olm_install_admin_console_doc/Documentation/olm-install.md

Two commits so that git sees the rename as a rename rather than a diff.

Github renders it as a diff changing the original file so probably best to view the rendered version or use the split diff viewer.

### CODE REVIEW

File: Documentation/manual-olm-install.md [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+## Install
+
+This will cover installing the metering-operator via the openshift-marketplace using `kubectl`/`oc` and will then create a Metering resource defining the configuration for the metering-operator to use to install the rest of the Metering stack.
+
+### Install Metering Operator
+
+Installing the metering-operator is done by creating a `Subscription` resource in a namespace with a `CatalogSource` containing the `metering` package.
+
+Currently we advise metering is installed into its own namespace which requires some setup.
+
```
Comment (EmilyM1) [inline Documentation/manual-olm-install.md]: Super minor: Currently we advise that ... or Currently we recommend that

**Comment by pruan-rht** [comment]
@chancez would adding screenshots be worth while here?

**Comment by chancez** [comment]
I asked @robszumski about that and he didn't think it should be necessary.

File: Documentation/manual-olm-install.md (hunk 1 above)
Comment (chancez) [reply Documentation/manual-olm-install.md]: Fair enough. That said, I didn't change this doc at all, I just renamed it, as I said in the PR summary. It just shows up as a diff.

File: Documentation/manual-olm-install.md [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+Once the subscription is created, OLM will create all the required resources for the metering-operator to run.
+This step takes a bit longer than others, within a few minutes the metering-operator pod should be created.
+Verify the metering-operator has been created and is running:
+
+```
+kubectl -n openshift-metering get pods
+NAME                                READY   STATUS    RESTARTS   AGE
+metering-operator-c7545d555-h5m6x   2/2     Running   0          32s
+metering-operators-bvpf7            1/1     Running   0          80s
```
Comment (pruan-rht) [inline Documentation/manual-olm-install.md]: @chancez when I try it locally, I don't see the `metering-operators-xxx` pod.  I only see the following

```
pruan@MacBook-Pro ~/workspace/testcases/OLM/tmp $ oc get pods                                                                                                                          
NAME                                 READY   STATUS    RESTARTS   AGE
metering-operator-7f85b548fd-dpmgh   2/2     Running   0          4m
```

Comment (chancez) [reply Documentation/manual-olm-install.md]: Yeah, that was supposed to be updated in a previous PR, Ill fix it here.

**Comment by tschuy** [comment]
the reason github isn't rendering it as a rename is because your first commit adds, not renames. Both files exist in commit 04b580b8: [olm-install.md](https://github.com/operator-framework/operator-metering/blob/04b580b89e48bab0b5b393a8aa10a2babb821362/Documentation/olm-install.md), [manual-olm-install.md](https://raw.githubusercontent.com/operator-framework/operator-metering/04b580b89e48bab0b5b393a8aa10a2babb821362/Documentation/manual-olm-install.md).

**Comment by tschuy** [comment]
this looks good to me, let me know when to /lgtm this.

**Comment by chancez** [comment]
/hold cancel

@tschuy can I get the lgtm?

**Pull request merged**